examples/integration/test/fixtures/bwb.xcodeproj/rules_xcodeproj/bazel/xcodeproj_extra_flags.bazelrc

```diff
@@ -1,4 +1,14 @@
-build:rules_xcodeproj --config=cache
-build:rules_xcodeproj_build --noverbose_failures
-build:rules_xcodeproj_indexbuild --noverbose_failures
-build:rules_xcodeproj_swiftuipreviews --noverbose_failures
+build:rules_xcodeproj_integration_build --config=rules_xcodeproj_build
+build:rules_xcodeproj_integration_generator --config=rules_xcodeproj_generator
+build:rules_xcodeproj_integration_indexbuild --config=rules_xcodeproj_indexbuild
+build:rules_xcodeproj_integration_info --config=rules_xcodeproj_info
+build:rules_xcodeproj_integration_swiftuipreviews --config=rules_xcodeproj_swiftuipreviews
+build:rules_xcodeproj_integration --config=cache
+build:rules_xcodeproj_integration_build --config=rules_xcodeproj_integration
```

Comment:
@jpsim Can you upload your version of this file? This shows how it should work as expected.

Reply:
```
build:my_config_build --config=rules_xcodeproj_build
build:my_config_generator --config=rules_xcodeproj_generator
build:my_config_indexbuild --config=rules_xcodeproj_indexbuild
build:my_config_info --config=rules_xcodeproj_info
build:my_config_swiftuipreviews --config=rules_xcodeproj_swiftuipreviews
```

Reply:
Thanks. I see the issue. I'll add a different integration test for this as well.

Reply:
@jpsim Can you give this another spin? I had to go another route to get the ordering and everything correct.

Reply:
Works now.